now dev failing

I just cloned the repo and am following the steps in CONTRIBUTIONS to start development using `now dev` but I get the following error:

```
Keslers-MBP-2:og-image keslert$ now dev -d
> [debug] [2019-06-29T16:53:19.314Z] Found config in file /Users/keslert/Projects/og-image/now.json
> [debug] [2019-06-29T16:53:19.319Z] Using Now CLI 15.6.2
> [debug] [2019-06-29T16:53:19.320Z] user supplied known subcommand
> Now CLI 15.6.2 dev (beta) — https://zeit.co/feedback/dev
> [debug] [2019-06-29T16:53:19.404Z] The yarn executable is already cached, not re-downloading
> [debug] [2019-06-29T16:53:19.405Z] Reading `now.json` file
> [debug] [2019-06-29T16:53:19.417Z] Locating files /Users/keslert/Projects/og-image
> [debug] [2019-06-29T16:53:19.420Z] Ignoring /Users/keslert/Projects/og-image/.git
> [debug] [2019-06-29T16:53:19.421Z] Ignoring /Users/keslert/Projects/og-image/.gitignore
> [debug] [2019-06-29T16:53:19.421Z] Ignoring /Users/keslert/Projects/og-image/CONTRIBUTING.md
> [debug] [2019-06-29T16:53:19.421Z] Ignoring /Users/keslert/Projects/og-image/Dockerfile
> [debug] [2019-06-29T16:53:19.421Z] Ignoring /Users/keslert/Projects/og-image/README.md
> [debug] [2019-06-29T16:53:19.421Z] Ignoring /Users/keslert/Projects/og-image/dist
> [debug] [2019-06-29T16:53:19.421Z] Ignoring /Users/keslert/Projects/og-image/node_modules
> [debug] [2019-06-29T16:53:19.428Z] Locating files /Users/keslert/Projects/og-image: 11.044ms
> [debug] [2019-06-29T16:53:19.436Z] Installing @now/build-utils@latest
> [debug] [2019-06-29T16:53:23.301Z] Locating files /Users/keslert/Projects/og-image
> [debug] [2019-06-29T16:53:23.301Z] Locating files /Users/keslert/Projects/og-image: 0.102ms
> [debug] [2019-06-29T16:53:23.302Z] Locating files /Users/keslert/Projects/og-image
> [debug] [2019-06-29T16:53:23.302Z] Locating files /Users/keslert/Projects/og-image: 0.059ms
> [debug] [2019-06-29T16:53:23.376Z] Locating files /Users/keslert/Projects/og-image
> [debug] [2019-06-29T16:53:23.376Z] Locating files /Users/keslert/Projects/og-image: 0.044ms
> [debug] [2019-06-29T16:53:23.727Z] Adding build match for "public/favicon.ico"
> [debug] [2019-06-29T16:53:23.727Z] Adding build match for "public/index.html"
> [debug] [2019-06-29T16:53:23.727Z] Adding build match for "public/robots.txt"
> [debug] [2019-06-29T16:53:23.727Z] Adding build match for "public/style.css"
> [debug] [2019-06-29T16:53:23.727Z] Adding build match for "public/tweet.png"
> [debug] [2019-06-29T16:53:23.727Z] Adding build match for "package.json"
> [debug] [2019-06-29T16:53:23.727Z] Adding build match for "src/card.ts"
> Setting up 1 Builder
> Building @now/static-build:package.json
> [debug] [2019-06-29T16:53:23.728Z] Using `@now/static-build@0.6.1`
> [debug] [2019-06-29T16:53:23.728Z] Creating build process for package.json
Downloading user files...
missing `engines` in `package.json`, using default range: 8.10.x
installing to /Users/keslert/Projects/og-image
yarn install v1.15.2
[1/4] Resolving packages...
success Already up-to-date.
Done in 0.09s.

> og-image@1.0.0 dev /Users/keslert/Projects/og-image
> now dev

internal/modules/cjs/loader.js:594
    throw err;
    ^

Error: Cannot find module '/Users/keslert/Projects/og-image/dev'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:592:15)
    at Function.Module._resolveFilename (pkg/prelude/bootstrap.js:1285:46)
    at Function.Module._load (internal/modules/cjs/loader.js:518:25)
    at Function.Module.runMain (pkg/prelude/bootstrap.js:1314:12)
    at startup (internal/bootstrap/node.js:274:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:608:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! og-image@1.0.0 dev: `now dev`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the og-image@1.0.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/keslert/.npm/_logs/2019-06-29T16_53_24_672Z-debug.log
```

The log file has the following error: 

```
0 info it worked if it ends with ok
1 verbose cli [ '/Users/keslert/Library/Caches/co.zeit.fun/runtimes/nodejs8.10/bin/node',
1 verbose cli   '/Users/keslert/Library/Caches/co.zeit.fun/runtimes/nodejs8.10/bin/npm',
1 verbose cli   'run',
1 verbose cli   'dev' ]
2 info using npm@5.6.0
3 info using node@v8.10.0
4 verbose run-script [ 'predev', 'dev', 'postdev' ]
5 info lifecycle og-image@1.0.0~predev: og-image@1.0.0
6 info lifecycle og-image@1.0.0~dev: og-image@1.0.0
7 verbose lifecycle og-image@1.0.0~dev: unsafe-perm in lifecycle true
8 verbose lifecycle og-image@1.0.0~dev: PATH: /Users/keslert/Library/Caches/co.zeit.fun/runtimes/nodejs8.10/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/Users/keslert/Projects/og-image/node_modules/.bin:/Users/keslert/Library/Caches/co.zeit.now/dev/builders:/Users/keslert/Library/Caches/co.zeit.fun/runtimes/nodejs8.10/bin:/Users/keslert/google-cloud-sdk/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
9 verbose lifecycle og-image@1.0.0~dev: CWD: /Users/keslert/Projects/og-image
10 silly lifecycle og-image@1.0.0~dev: Args: [ '-c', 'now dev' ]
11 silly lifecycle og-image@1.0.0~dev: Returned: code: 1  signal: null
12 info lifecycle og-image@1.0.0~dev: Failed to exec dev script
13 verbose stack Error: og-image@1.0.0 dev: `now dev`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (/Users/keslert/Library/Caches/co.zeit.fun/runtimes/nodejs8.10/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:285:16)
13 verbose stack     at emitTwo (events.js:126:13)
13 verbose stack     at EventEmitter.emit (events.js:214:7)
13 verbose stack     at ChildProcess.<anonymous> (/Users/keslert/Library/Caches/co.zeit.fun/runtimes/nodejs8.10/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at emitTwo (events.js:126:13)
13 verbose stack     at ChildProcess.emit (events.js:214:7)
13 verbose stack     at maybeClose (internal/child_process.js:925:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
14 verbose pkgid og-image@1.0.0
15 verbose cwd /Users/keslert/Projects/og-image
16 verbose Darwin 18.0.0
17 verbose argv "/Users/keslert/Library/Caches/co.zeit.fun/runtimes/nodejs8.10/bin/node" "/Users/keslert/Library/Caches/co.zeit.fun/runtimes/nodejs8.10/bin/npm" "run" "dev"
18 verbose node v8.10.0
19 verbose npm  v5.6.0
20 error code ELIFECYCLE
21 error errno 1
22 error og-image@1.0.0 dev: `now dev`
22 error Exit status 1
23 error Failed at the og-image@1.0.0 dev script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

I've tested `now dev` on a few other example projects and it is working just fine.

--- a/package.json
+++ b/package.json
@@ -4,7 +4,6 @@
   "description": "Generate an open graph image for twitter/facebook/etc",
   "main": "dist/card.js",
   "scripts": {
-    "dev": "now dev",
     "build": "tsc",
     "now-build": "tsc",
     "watch": "tsc --watch"

